src/self-update/README.md

```diff
@@ -0,0 +1,66 @@
+# Introduction
+
+That document specifies the feature self-update for thin-edge.
+
+The self-update allows a Cloud Operator to update one thin-edge version that is installed on a device to a newer version, via the Cloud connection. The feature is referred as self-update cause that update process is managed by thin-edge own's Software Management.
+
+# Requirements
+
+* Be default self-update shall use Debian Package Manager APT.
+  * NOTE: APT Package Manager provides a very basic rollback mechanism that still can fail (e.g. due to some unexpected device state or power-loss).
```

Comment:
For me this is not a requirement but the proposed solution.

Reply:
Now it's moved to section "Design Principles".